mleap-runtime/src/main/scala/ml/combust/mleap/bundle/ops/clustering/LDAModelOp.scala

```diff
@@ -0,0 +1,86 @@
+package ml.combust.mleap.bundle.ops.clustering
+
+import ml.combust.bundle.BundleContext
+import ml.combust.bundle.dsl._
+import ml.combust.bundle.op.{OpModel, OpNode}
+import ml.combust.mleap.runtime.MleapContext
+import ml.combust.mleap.tensor.DenseTensor
+
+/**
+  * Created by mageswarand on 3/3/17.
+  *
+  * https://github.com/combust/mleap/blob/master/mleap-runtime/src/main/scala/ml/combust/mleap/bundle/ops/classification/LogisticRegressionOp.scala
+  *
+  */
+class LDAModelOp extends OpNode[MleapContext, LDAModel, LocalLDAModel] {
+
+  override val Model: OpModel[MleapContext, LocalLDAModel] = new OpModel[MleapContext, LocalLDAModel] {
+    override val klazz: Class[LocalLDAModel] = classOf[LocalLDAModel]
+
+    override def opName: String = "lda_local_model_op"
+
+    override def store(model: Model, obj: LocalLDAModel)(implicit context: BundleContext[MleapContext]): Model = {
+      val topicMatrixArray = obj.topicsMatrix.toDenseMatrix.toArray
+      val topicMatrixRows = obj.topicsMatrix.rows
+      val topicMatrixCols = obj.topicsMatrix.cols
+
+      println("Modle Paramaters during MLEAP saving...")
+      println("Rows: " + topicMatrixRows)
+      println("Cols: " + topicMatrixCols)
+      println("Array: " + topicMatrixArray.size)
+
```

Comment:
We are able to see the print coming up until this line and then the pipeline loading fails